buildrunner/utils.py

```diff
@@ -163,6 +164,30 @@ def hash_sha1(file_name_globs=None):
     return hasher.hexdigest()
 
 
+def logger(results_dir, colorize):
+    """
+    create the log file and open for writing
+    """
+    log_file = None
+    try:
+        os.makedirs(results_dir)
+    except OSError as exc:
+        if exc.errno != errno.EEXIST:
+            sys.stderr.write(f'ERROR: {str(exc)}\n')
+            sys.exit(os.EX_UNAVAILABLE)
+
+    try:
+        log_file_path = os.path.join(results_dir, 'build.log')
+        log_file = open(log_file_path, 'w', encoding='utf-8')
```

Comment:
Does this log file ever get closed? I am still looking through the code so maybe I will see it as I continue.

Reply:
It does in `__init__.py`, but not in `config.py`